src/synapse/azext_synapse/_params.py

```diff
@@ -43,12 +39,20 @@ def load_arguments(self, _):
                        help='The size of the returned list.By default it is 20 and that is the maximum.')
 
     with self.argument_context('synapse spark batch create') as c:
-        c.argument('language', arg_type=get_enum_type(['SCALA', 'PYTHON', 'DOTNET'], default='SCALA'),
+        c.argument('main_definition_file', help='The main file used for the job.')
+        c.argument('main_class_name',
+                   help='The fully-qualified identifier or the main class that is in the main definition file.')
+        c.argument('command_line_arguments', nargs='+',
+                   help='Optional arguments to the job (Note: please use storage URIs for file arguments).')
+        c.argument('archives', nargs='+', help='The array of archives.')
+
+    with self.argument_context('synapse spark batch create') as c:
+        c.argument('language', arg_type=get_enum_type(SparkBatchLanguage, default=SparkBatchLanguage.Scala),
                    help='The spark batch job language.')

```

Comment:
```suggestion
                   help='The Spark batch job language.')
```

Reply:
Same comments with other occurrences of "Spark"

Reply:
fixed

---
In reply to: [418070764](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r418070764) [](ancestors = 418070764)

Reply:
fixed

---
In reply to: [418071965](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r418071965) [](ancestors = 418071965)